packages/core/src/Stores/client-store.js

```diff
@@ -988,7 +988,7 @@ export default class ClientStore extends BaseStore {
         }
     };
 
-     getIsMarketTypeMatching = (account, market_type) => {
+    getIsMarketTypeMatching = (account, market_type) => {
```

Comment:
Prettier made this fix since it seems like the last person that made changes to this function did not format his/her code.